code/_onclick/drag_drop.dm

```diff
@@ -21,10 +21,26 @@
 
 
 /client/MouseDown(object, location, control, params)
+	if(!control)
+		return
 	if(mouse_down_icon)
 		mouse_pointer_icon = mouse_down_icon
+	if(SEND_SIGNAL(src, COMSIG_CLIENT_MOUSEDOWN, object, location, params) & COMSIG_CLIENT_MOUSEDOWN_INTERCEPT)
```

Comment:
Provide all relevant arguments for new signals you create even if you wont use it yourself